src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -216,6 +222,22 @@ class BinaryCompactObject : public DomainCreator<3> {
     static type default_value() { return true; }
   };
 
+  struct UseLogarithmicMapOuterSphericalShell {
+    using type = bool;
+    static constexpr OptionString help = {
+        "Use a logarithmically spaced radial grid in Layer 5, the outer "
+        "spherical shell."};
+    static type default_value() noexcept { return false; }
+  };
+
+  struct OuterLayerRadialRefinements {
+    using type = size_t;
+    static constexpr OptionString help = {
+        "Radially refine Layer 5, the outer spherical shell, this many times."};
```

Comment:
I don't understand what "this many times" means. Is this the radial refinement level or not? If it is I would suggest "Radial refinement level of Layer 5, the outer spherical shell that covers the wave zone."

Reply:
Changed to "Addition to radial refinement level in Layer 5 (the outer spherical shell that covers that wave zone), beyond the refinement level set by InitialRefinement.